kotlin-eclipse-core/src/org/jetbrains/kotlin/core/compiler/KotlinCompiler.java

```diff
@@ -106,7 +106,14 @@ public KotlinCompilerResult execKotlinCompiler(@NotNull String[] arguments) {
         List<String> command = new ArrayList<>();
         command.add("-kotlin-home");
         command.add(ProjectUtils.getKtHome());
-        command.add("-no-jdk");
+
+        boolean jdkHomeUndefined = kotlinProperties.isJDKHomDefined();

```

Comment:
I think the names here are confusing. Are those properties `true` when jdk home is defined or undefined?